src/main/scala/mesosphere/marathon/api/akkahttp/v2/AppsController.scala

```diff
@@ -105,7 +106,8 @@ class AppsController(
           reject(Rejections.EntityNotFound.app(appId))
         case Some(info) =>
           authorized(ViewResource, info.app).apply {
-            complete(Json.obj("app" -> info))
+            val ramlAppInfo = raml.AppInfo(app = raml.Raml.toRaml(info.app))
```

Comment:
`raml.Raml.toRaml` is obviously not nice. However, I am really in favour of being explicit here. I don't see a great benefit in `info.app.toRaml`.